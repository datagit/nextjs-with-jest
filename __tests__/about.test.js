import React from 'react'
import { render } from '@testing-library/react'
// for toMatchSnapshot
import renderer from 'react-test-renderer'
import About from '../pages/about'


test('renders page about', () => {
  const { getByText } = render(<About />)
  const linkElement = getByText(
    /this is about/
  )
  expect(linkElement).toBeInTheDocument()
})


it('renders about unchanged', () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
  })