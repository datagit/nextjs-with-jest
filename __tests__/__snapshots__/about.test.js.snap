// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<div
  className="container"
>
  <h1>
    this is about
  </h1>
</div>
`;
